man/man8/zfs.8

```diff
@@ -3607,9 +3607,9 @@ The command can also be given a path to a ZFS file system shared on the system.
 .It Xo
 .Nm
 .Cm bookmark
-.Ar snapshot bookmark
+.Ar snapshot Ns | Ns Ar bookmark bookmark
```

Comment:
I take you mean "Origin bookmark" and "target bookmark" here for the copy function?
Maybe clear that up, instead of 2 times "bookmark"?